src/components/com_tjdashboard/administrator/tables/widgets.php

```diff
@@ -55,4 +55,45 @@ public function check()
 
 		return parent::check();
 	}
+
+	/**
+	 * Method to delete a Widget.
+	 *
+	 * @param   mixed  $pk  Primary key value to delete. Optional
+	 * 
+	 * @return  boolean  True on success.
+	 *
+	 * @since   1.0.0
+	 */
+	public function delete($pk = null)
+	{
+		JSession::checkToken() or jexit(JText::_('JINVALID_TOKEN'));
+		$result = 0;
+
+		if (!empty($pk))
+		{
+			try
+			{
+				$db = JFactory::getDbo();
```

Comment:
@shitalkakade24 Load the widgets from library

Reply:
done